avm/res/network/nat-gateway/tests/e2e/existingPip/main.test.bicep

```diff
@@ -52,9 +50,8 @@ module testDeployment '../../../main.bicep' = [
     scope: resourceGroup
     name: '${uniqueString(deployment().name, resourceLocation)}-test-${serviceShort}-${iteration}'
     params: {
-      location: resourceLocation
       name: '${namePrefix}${serviceShort}001'
-      zone: 1
+      availabilityZone: -1
```

Comment:
why disabling zones here?

Reply:
To test different settings. This test (existingPip) is not about testing if availabilityZones work. However it's now required, so I have to set something.

Reply:
Sounds good. It was set to 1 before the rename, hence asked. Feel free to resolve